source/_docs/guides/composer_convert.md

```diff
@@ -0,0 +1,118 @@
+
+### Custom Code
+#### Modules and Themes
+
+Custom code should be manually copied from the existing site repository.
+
+module:
+```sh
+me@my-computer:~/Sites/my-site(composify) $ cp -r /modules/custom/awesome_module ../my-site-composer/web/modules/custom
+```
+theme:
+```sh
+me@my-computer:~/Sites/my-site(composify) $ cp -r /themes/custom/great_theme ../my-site-composer/web/themes/custom
+```
+
+Follow suit with any other custom code you need to carry over.
+
+#### Settings.php
+Your existing site may have customizations to `settings.php` or any other config files. Review these carefully and extract relavent changes from these files to copy over, always review any file paths referenced in the code as these may change in the transition. It is not wise to completely overwrite the file with the old one as there are customizations for moving the configuration directory you don't want to overwrite.
+
+#### Configuration
+The preferred (and assumed) location of the configuration directories when using a nested docroot and Composer is at the root of the repository next to the web directory:
+```
+my-site-composer
+|-web
+|-config    <--Here!
+|-vendor
+|-composer.json
+|-etc...
+```
+
+Locate the configuration files in your existing site and move them here. If they are stored in the files directory on your existing site, you will want to retreive them via SFTP (as the Git clone would not contain them).
+
+## Prepare to Deploy 	&#128640;
+
+At this point, your new project directory should contain all of the unique code from your existing Drupal 8 site plus all of the code required to make a Composer driven project work. Since Pantheon requires all runtime code to be present when deploying to the platform, if no CI solution is a part of your workflow, you must now modify the project to be deployable straight to Pantheon. From the `my-site-composer` directory, run the following:
+
+```sh
+composer prepare-for-pantheon
+```
+
+This should modify the `.gitignore` file and cleanup an errant `.git` directories in the codebase to prepare your new code for direct deployment to Pantheon.
+
+## Commit
+We now have to commit all this work to our Pantheon repo. From the `my-site` directory, run the following:
+
+```sh
+cp -r .git ../my-site-composer 
+cd ../my-site-composer
+git add .
+git commit -m "Convert to Composer Based install
+```
+
+You should see a large amount of files committed to the new branch we created earlier.
+
+## Deploy &#128640;
+
+Now that we've committed the code on a branch, if the site has multi-dev, you can deploy that branch directly to a new multidev and test the site in the browser. If there are any issues, utilize your site's logs via `terminus drush my-site.composify -- wd-show` to inspect the watchdog logs, or follow the directions on (our documentation on log collect)[/docs/logs]
```

Comment:
There's a caution here. If the original site did not have a relocated document root (and it almost certainly will not), then creating a multidev site from an existing branch (as created and pushed above) with a pantheon.yml that DOES contain a relocated document root (as the example project does), then a bug will prevent the docroot from being relocated. The user should create the multidev first, and then push the new code with the new pantheon.yml + relocated docroot as a second step. If that is done, then the docroot will relocate correctly.

If the user ends up with a broken site (created a multidev with an existing commit, so docroot did not move), then adding a change to pantheon.yml (e.g. add a comment) and pushing it to the multidev will fix things up.

Reply:
Addressed above in new revision